subprojects/platform-play/src/main/java/org/gradle/play/internal/platform/PlayMajorVersion.java

```diff
@@ -60,6 +61,9 @@ public static PlayMajorVersion forPlatform(PlayPlatform targetPlatform) {
     public static PlayMajorVersion forPlayVersion(String playVersion) {
         VersionNumber versionNumber = VersionNumber.parse(playVersion);
         if (versionNumber.getMajor() == 2) {
+            if (versionNumber.getMinor() == 2) {
```

Comment:
Shouldn't the expression cover older versions than 2.2 as well by using `versionNumber.getMinor() <= 2`?

Reply:
If version is `1.x`, it will fall back to `invalidVersion()`, which throws an exception. So I think it's okay here.

Reply:
What about version 2.1?

Reply:
We only have `PLAY_2_2_X`~`PLAY_2_6_X` - all other values will fall into `invalidVersion()`.

Reply:
I see. OK, thanks.